doc/build.md

```diff
@@ -0,0 +1,191 @@
+# Build & Installation
+
+1. Download and install CMake:
+
+   VCML uses [CMake](https://cmake.org/) as its building system. Please install it from the repository of your
+   distribution or download ready-to-use executables from CMake's [download page](https://cmake.org/download/).
+
+2. Clone VCML repository and initialize submodules:
+
+    ```sh
+    git clone https://github.com/machineware-gmbh/vcml.git --recursive
+    ```
+    or, alternatively
+    ```sh
+    git clone https://github.com/machineware-gmbh/vcml.git
+    git submodule update --init
+    ```
+
+3. *Optional*: Download and build SystemC.
```

Comment:
Mention that `libmwr` is also an optional dependency which will be automatically cloned from github or searched for at `$MWR_HOME` (shoud point to the souce toplevel dir where the CMakeLists.txt is)